README.docker.md

```diff
@@ -0,0 +1,17 @@
+# Tsugite in the browser
+
+Debian with Mesa libraries and Gallium drivers "llvmpipe" and "softpipe" provides OpenGL support inside a Docker container without the need for a GPU.
+
+Should therefore run on most hardware configurations, albeit slower than when utilizing hardware with GPUs.
+
+A web UI based on noVNC exposes the python app for usage from a web browser.
+
+
+# Usage
+
+After a local build, use:
+
+		docker run --rm -port "8083:8083" tsugite 
```

Comment:
```suggestion
		docker run --rm --expose "8083:8083" tsugite 
```

Or `-p`, according to [the docs](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose) ;-)

Reply:
-p it is :)

Reply:
Fix in b5332ff